genie-agent/src/test/groovy/com/netflix/genie/agent/configs/AwsAutoConfigurationSpec.groovy

```diff
@@ -25,23 +25,26 @@ import com.amazonaws.regions.Regions
 import com.netflix.genie.agent.aws.s3.S3ClientFactory
 import com.netflix.genie.agent.execution.services.impl.NoOpArchivalServiceImpl
 import com.netflix.genie.agent.execution.services.impl.S3ArchivalServiceImpl
+import org.springframework.cloud.aws.autoconfigure.context.properties.AwsRegionProperties
 import spock.lang.Specification
 import spock.lang.Unroll
 
 /**
  * Specifications for {@link AwsAutoConfiguration}.
  *
- * @author tgianos
- * @since 4.0.0
+ * @author tgianos* @since 4.0.0
```

Comment:
ugh. Something happened with my new version of Intellij (2018.3) where the groovy file java docs are auto formatting like this not matter what I seem to do. I've just been removing the `@since` to avoid this problem. Annoying not sure if there's something weird going on with intellij

Reply:
Fixed